### PR TITLE
*: s/the the /the /

### PR DESCRIPTION
**Pull request by jordanlewis**
This is a surprisingly common typo!

Release note: None

### CODE REVIEW

**Review by rytaft — APPROVED** [review]
:lgtm: 

> Reviewed 54 of 54 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57225#-:-MNOu3-DAWcutESWtnxP:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained

File: pkg/ccl/backupccl/targets.go [hunk 1]
```diff
@@ -485,7 +485,7 @@ func descriptorsMatchingTargets(
 // descriptor revision matches if it is an earlier revision of a descriptor in
 // descs (same ID) or has parentID in `expanded`. Deleted descriptors are
 // represented as nil. Fills in the `priorIDs` map in the process, which maps
-// a descriptor the the ID by which it was previously known (e.g pre-TRUNCATE).
```
Comment (pbardea) [inline pkg/ccl/backupccl/targets.go:488]: I think that the typo here is that this should be `to the`, but that can also be fixed in a follow-up to keep this change mechanical. :)

**Review by RaduBerinde — COMMENTED** [review]
Thanks for the the fix.

**Pull request merged**